openshift-ci/Dockerfile

```diff
@@ -57,7 +57,7 @@ ADD --chmod=755 run.sh oc_login.sh /tmp/tests/
 # to debug on local
 # ADD --chmod=755 oc_login_local.sh /tmp/tests/oc_login.sh
 
-USER jboss
+USER default
```

Comment:
@tveskrna Is this change somewhere documented ? Or why is it needed ?

Reply:
jboss username doesn't exist in ubi9, just found during testing, same rights has "default" user